app/data/0_rogue_one.json

```diff
@@ -364,8 +366,21 @@
                     "title": ""
                 }
             ],
-            "id": "82504ab4-de5d-41fd-9a86-9ad5ea483296",
+            "id": "rogue-one-group",
             "title": ""
+        },
+        {
+            "blocks": [
+                {
+                    "type": "summary",
+                    "id": "summary",
+                    "title": ""
```

Comment:
Blocks according to the schema definition don't need titles, we might as well remove all the summary titles if they are going to be blank